### DISCUSSION TITLE
NextJs 15 + Turbo + Expo Setup

### DISCUSSION THREAD

**Discussion by shawnmclean** [nativewind/nativewind#946] [Q&A]
Hey guys,

I'm trying to integrate NativeWind with the latest turbo repo template.

I'm stuck at the react native part, but I hope to find some help from the NativeWind community.

Following the documents, it offloads the troubleshooting of RNW, which I cannot find good docs to implement this on NextJs.

I'm just looking for a simple repo where there is a pure nextjs 14/15 and expo app, sharing RN components from a shared lib.

No routing, solito, etc.

Can anyone help guide me on this setup? My repo: https://github.com/shawnmclean/sovoli

I'd also like this to serve as a template and update for the main docs site, in which I can help with.

To start: `pnpm dlx create-turbo@latest`

Or if anyone can get [this ](https://github.com/vercel/turbo/tree/main/examples/with-react-native-web)to work as they have a template thats old and doesn't seem to work with Expo 51.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
